R/experiments.R

```diff
@@ -70,21 +70,24 @@ create_experiment <- function(name, artifact_location = "", client = mlflow_clie
 #' @importFrom purrr reduce
 #' @importFrom tibble as_tibble
 #'
+#' @param max_results Maximum number of experiments to retrieve.
```

Comment:
the new parameter is also a breaking change. the `list` endpoint didn't seem to have this